matlab/test/Ice/objects/CB8I.m

```diff
@@ -1,5 +1,5 @@
 %
-% Copyright (c) ZeroC, Inc. All rights reserved.
+% Copyright (c) ZeroC, Inc.
 %
```

Comment:
And the MATLAB files still have a leading/trailing blank line around the copyright.
Maybe that's 'expected' in MATLAB though. Idk.